runtime/jvmti/CMakeLists.txt

```diff
@@ -79,6 +79,8 @@ omr_add_exports(j9jvmti
 	J9VMDllMain
 )
 
+target_enable_ddr(j9jvmti j9ddr)
```

Comment:
Why does the macro take both the target lib and `j9ddr`?

Reply:
the cmake ddr support has the concepts of "ddr sets" which are basically the output of ddrgen. The idea being that you might want multiple independent ddr blobs (eg the omr ddr tests creates its own ddr set https://github.com/eclipse/omr/blob/65f81a4e71d7b8a852be7fc577793e8753b80a9b/ddr/test/CMakeLists.txt#L33)